docs/versions1.json

```diff
@@ -1,6 +1,11 @@
 [
     {
         "preferred": true,
+        "version": "25.09",
+        "url": "../25.09"
+    },
+    {
+        "preferred": false,
         "version": "0.25.7",
         "url": "../0.25.7"
```

Comment:
Can this version be renamed to 25.07?

Reply:
yes i plan to fix this but it's a kinda slow process because i need to download the old docs and re-upload them to a different s3 directory